docs/tutorials/mobile/pose-detection.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Object detection and pose estimation with Yolov8
+description: Run the Yolov8 model on mobile (iOS and Android) with built-in pre and post processing to perform object detection and pose estimation
+image: /images/pose-thumbnail.png
+parent: Deploy on mobile
+grand_parent: Tutorials
+nav_order: 1
+---
+
+# Object detection and pose estimation on mobile with Yolov8
+
+Learn how to build and run ONNX models with built-in pre and post processing for object detection and pose detection.
+
+## Object detection with Yolov8
+
+You can find the full source code for the [Android](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/mobile/examples/object_detection/ios) app in the ONNX Runtime inference examples repository.
+
+### Build the ONNX model with built-in pre and post processing
+
+This step is optional as the model is available in the examples repository in the applications folders above. If you are interested, the following steps show you how to build the model yourself.
+
+Create a Python environment and install the following packages.
+
+```bash
+pip install --upgrade onnx onnxruntime onnxruntime-extensions pillow
+```
+
+Download the following script to build the model.
+
+```bash
+curl https://raw.githubusercontent.com/microsoft/onnxruntime-extensions/main/tutorials/yolo_e2e.py > yolo_e2e.py
+```
+
+Run the script.
+
+```bash
+python yolo_e2e.py --test_image <image to test on>
+```
+
+After the script has run, you will see one PyTorch model and two ONNX models:
+* `yolov8n.pt`: The original Yolov8 PyTorch model
+* `yolov8n.onnx`: The exported Yolov8 ONNX model
+* `yolov8n.with_pre_post_processing.onnx`: The ONNX model with pre and post processing included in the model
+* `<test image>.out`: Your test image with bounding boxes supplied.
```

Comment:
The post processing creates a jpg so can we use that as the output filename suffix? '.out' isn't going to open by default in an image viewer.